src/containers/Sidebar.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import SidebarMenu from "../components/SidebarMenu";
+import SidebarSubMenu from "../components/SidebarSubMenu"
+import SidebarItemMenu from "../components/SidebarItemMenu"
+
+const Sidebar = () => {
+
+    return(
+  <>
+            <SidebarMenu>
```

Comment:
Can you put values in array and make a .map ?

Reply:
😭 yes ! But not now ..